espnet2/bin/asr_inference.py

```diff
@@ -347,7 +360,9 @@ def __init__(
         if bpemodel not in ["whisper_en", "whisper_multilingual"]:
             converter = TokenIDConverter(token_list=token_list)
         else:
-            converter = OpenAIWhisperTokenIDConverter(model_type=bpemodel)
+            converter = OpenAIWhisperTokenIDConverter(
+                model_type=bpemodel, language=tokenizer_language
```

Comment:
Can we specify any language id here or only the langs supported in Whisper model?